test/DotNetIsolated60/DotNetIsolated60.csproj

```diff
@@ -6,13 +6,15 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <FunctionsEnableWorkerIndexing>True</FunctionsEnableWorkerIndexing>
+    <FunctionsAutoRegisterGeneratedMetadataProvider>True</FunctionsAutoRegisterGeneratedMetadataProvider>
+    <FunctionsEnableExecutorSourceGen>True</FunctionsEnableExecutorSourceGen>
```

Comment:
Added these msbuild properties for auto registration and removed the explicit extension method calls from Program.cs